week-7/learning-outcomes.md

```diff
@@ -0,0 +1,47 @@
+# Week 7 Learning Outcomes
+
+## REST APIs
+
+### HTTP & REST
+
+- [ ] **We can build an API that returns JSON**
+- [ ] We can build an API that responds to all HTTP methods
+- [ ] We set relevant HTTP headers on our API responses
+- [ ] We can explain what makes a service RESTful
+
+### Server design
+
+- [ ] **We can structure our server code effectively**
+- [ ] We can structure our code to match REST methods
+
+### Express
+
+- [ ] **We can use Express to build an HTTP server**
+- [ ] We can serve our static assets using Express
+- [ ] We can use Express to handle requests for different routes
+- [ ] We can explain what Express middleware is
+- [ ] We can use 3rd party middleware
```

Comment:
Is it worth being more specific?, e.g. *"We can use at least three pieces of third party middleware in our app"*.

Reply:
I wouldn't want to give them the idea that there's a minimum amount of 3rd party stuff they should be using. I think I originally had more specific outcomes like "can use middleware to handle errors", "can use middleware to parse POST bodies" etc. That might be a good option